benefits/core/models.py

```diff
@@ -419,3 +421,30 @@ def for_user(user: User):
 
         # the loop above returns the first match found. Return None if no match was found.
         return None
+
+
+class EnrollmentMethods:
+    DIGITAL = "digital"
+    IN_PERSON = "in_person"
+
+
+class EnrollmentEvent(models.Model):
+    """A record of a successful enrollment."""
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4)
```

Comment:
I guess it probably shouldn't be editable even locally, right?

Reply:
Yeah, I think setting `editable=False` for it would be good

Reply:
Thanks for that suggestion. I tried both `editable=False` on the model, which does make the field read-only, because it hides it entirely from the Admin UI!

I went with this approach using the ModelAdmin instead, so that the field still shows up in the Admin UI (now it's at the bottom, which is probably better too).